components/cloud-foundry/test/e2e/other/applications.services.spec.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+  'use strict';
+
+  // var helpers = require('../../../../app-core/frontend/test/e2e/po/helpers.po');
+  // var resetTo = require('../../../../app-core/frontend/test/e2e/po/resets.po');
+  // var loginPage = require('../../../../app-core/frontend/test/e2e/po/login-page.po');
+  var galleryPage = require('../po/applications/applications.po');
+  // var applicationPage = require('../po/applications/application.po');
+
+  xdescribe('Application - Services', function () {
```

Comment:
xdescribe - is that because we don't have any services? We should make this configurable and we able to skip based on services being present or not.